blueprints/codes/latex_formula.py

```diff
@@ -2,6 +2,69 @@
 from dataclasses import dataclass
 
 
+def value_to_latex_text(value: str | float) -> str:
```

Comment:
what about value to latex only? Or to_latex_text? 
To keep it short? Or would _text be more specific? As this is a function that will be used a lot it might be usefull to keep it as short (but also readable) as possible.

Reply:
maybe let all functions have the same structure like:
* latex_text
* latex_max_curly_brackets
* latex_fraction

Or another syntax, but now all three functions have a different structure.